### PR TITLE
fix spelling errors

### PR DESCRIPTION
**Pull request by askeblad**
lsp-test/src/Language/LSP/Test.hs
lsp/src/Language/LSP/Server/Core.hs
lsp/src/Language/LSP/VFS.hs
lsp/test/URIFilePathSpec.hs

@michaelpj missed these in `#413`

### CODE REVIEW

File: lsp-test/src/Language/LSP/Test.hs [hunk 1]
```diff
@@ -241,7 +241,7 @@ runSessionWithHandles' serverProc serverIn serverOut config' caps rootDir sessio
   exitServer :: Session ()
   exitServer = request_ SShutdown Empty >> sendNotification SExit Empty
 
-  -- | Listens to the server output until the shutdown ack,
+  -- | Listens to the server output until the shutdown hack,
```
Comment (michaelpj) [inline lsp-test/src/Language/LSP/Test.hs:244]: This is wrong, it was "ack" as in "acknowledges"

**Review by michaelpj — APPROVED** [review]

File: lsp-test/src/Language/LSP/Test.hs (hunk 1 above)
Comment (askeblad) [reply lsp-test/src/Language/LSP/Test.hs:244]: could it be typed as `SHUTDOWN ACK`

Comment (michaelpj) [reply lsp-test/src/Language/LSP/Test.hs:244]: I think it's clear enough to me, but I wouldn't object to ACK.

Comment (michaelpj) [reply lsp-test/src/Language/LSP/Test.hs:244]: You can just push more commits to your branch, and it will update the PR.

Comment (askeblad) [reply lsp-test/src/Language/LSP/Test.hs:244]: don't you have to squash

Comment (michaelpj) [reply lsp-test/src/Language/LSP/Test.hs]: No.

**Comment by michaelpj** [comment]
Probably "lossily"

**Comment by askeblad** [comment]
> Probably "lossily"

would spj concur

then I would submit a PR for the cabal repo with that change

**Comment by askeblad** [comment]
> Probably "lossily"

thanks

opened a new cabal repo PR

**Pull request merged**